src/modules/rlm_mschap/rlm_mschap.c

```diff
@@ -1999,7 +1999,7 @@ static rlm_rcode_t CC_HINT(nonnull) mod_authenticate(void *instance, REQUEST *re
 			if (inst->with_ntdomain_hack) {
 				username_string++;
 			} else {
-				RWDEBUG2("NT Domain delimiter found, should with_ntdomain_hack of been enabled?");
+				RWDEBUG2("NT Domain delimiter found, should we have enabled with_ntdomain_hack?");
```

Comment:
Surely just correcting to "should with_ntdomain_hack have been enabled," rather than "should we have enabled..." - which implies that the FR authors got it wrong...